runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1708,7 +1797,7 @@ static void buildInterfaceCall(TR::CodeGenerator *cg, TR::Node *callNode, TR::Re
    if (comp->getOptions()->enableDebugCounters())
```

Comment:
This function calls `enableDebugCounters()` four times.  I would suggest introducing a new variable for that.
```
bool enableDebugCounters = comp->getOptions()->enableDebugCounters();
```

Reply:
Introduced a new variable `enableDebugCounters` as suggested.